packages/shared/sdk-client/src/api/LDIdentifyOptions.ts

```diff
@@ -33,4 +33,15 @@ export interface LDIdentifyOptions {
    * not be required.
    */
   noTimeout?: boolean;
+
+  /**
+   * If true, the identify operation will be sheddable. This means that if multiple identify operations are done, without
```

Comment:
```suggestion
   * If true, the identify operation will be sheddable. This means that if multiple identify operations are started without
```

The word 'done' could be confused with finished.